src/PowerSystems2PRAS.jl

```diff
@@ -1,211 +1,97 @@
+            PSY.add_supplemental_attribute!(sys, comp, transition_data)
+        end
     end
-    # add N to S2P_metadata object
-    add_N!(s2p_meta)
+end
 
-    # TODO: Is it okay to just get the first elemnt of vector returned by PSY.get_time_series_resolutions?
-    sys_res =
-        round(Dates.Millisecond(first(PSY.get_time_series_resolutions(sys))), Dates.Hour)
-    if iszero(sys_res.value)
-        sys_res = round(
-            Dates.Millisecond(first(PSY.get_time_series_resolutions(sys))),
-            Dates.Minute,
-        )
-        s2p_meta.pras_resolution = Dates.Minute
-    end
-    s2p_meta.pras_timestep = sys_res.value
-    start_datetime_tz = TimeZones.ZonedDateTime(s2p_meta.first_timestamp, TimeZones.tz"UTC")
-    finish_datetime_tz =
-        start_datetime_tz + s2p_meta.pras_resolution((s2p_meta.N - 1) * sys_res)
-    my_timestamps =
-        StepRange(start_datetime_tz, s2p_meta.pras_resolution(sys_res), finish_datetime_tz)
+"""
+    $(TYPEDSIGNATURES)
 
-    @info "The first timestamp of PRAS System being built is : $(start_datetime_tz) and last timestamp is : $(finish_datetime_tz) "
-    #######################################################
-    # Ensure no double counting of HybridSystem subcomponents
-    # TODO: Not sure if we need this anymore.
-    #######################################################
-    dup_uuids = Base.UUID[]
-    h_s_comps = PSY.get_available_components(PSY.HybridSystem, sys)
-    for h_s in h_s_comps
-        push!(dup_uuids, PSY.IS.get_uuid.(PSY._get_components(h_s))...)
-    end
-    # Add HybridSystem sub component UUIDs to s2p_meta
-    if ~(isempty(dup_uuids))
-        s2p_meta.hs_uuids = dup_uuids
-    end
-    #######################################################
-    # PRAS Regions - Areas in PowerSystems.jl
-    #######################################################
-    @info "Processing $(aggregation) objects in Sienna/Data PowerSystems System... "
-    regions = collect(PSY.get_components(aggregation, sys))
-    if ~(length(regions) == 0)
-        @info "The Sienna/Data PowerSystems System has $(length(regions)) regions based on PSY AggregationTopology : $(aggregation)."
-    else
-        error(
-            "No regions in the Sienna/Data PowerSystems System. Cannot proceed with the process of making a PRAS SystemModel.",
-        )
-    end
+Extract region load as a matrix of Int64 values.
 
-    region_names = PSY.get_name.(regions)
-    num_regions = length(region_names)
+Uses to all StaticLoad objects.
 
-    region_load = Array{Int64, 2}(undef, num_regions, s2p_meta.N)
+Note that behavior is not controlled by a formulation.
+"""
+function get_region_loads(sys::PSY.System, s2p_meta::S2P_metadata, regions)
+    region_load = Array{Int64, 2}(undef, length(regions), s2p_meta.N)
 
+    # FIXME
     for (idx, region) in enumerate(regions)
         reg_load_comps =
             get_available_components_in_aggregation_topology(PSY.StaticLoad, sys, region)
```

Comment:
Be aware that a system can contain other load types. This should probably support all subtypes of `ElectricLoad`

Reply:
I did not plan on messing with load or lines yet #61